packages/dashboards/package.json

```diff
@@ -71,9 +71,10 @@
     "ember-tag-input": "^1.2.1",
     "ember-tether": "^0.4.1",
     "ember-toggle": "5.2.1",
-    "ember-tooltips": "^2.9.2",
+    "ember-tooltips": "^2.10.0",
     "ember-truth-helpers": "^2.0.0",
     "ember-uuid": "^1.0.0",
+    "klaw": "^3.0.0",
```

Comment:
What is this needed for?

Reply:
When I ran dashboards before, it told me it was missing that package, but it doesn't seem to need it now 🤔